turbinia/workers/__init__.py

```diff
@@ -690,13 +691,21 @@ def validate_result(self, result):
 
     if isinstance(result, TurbiniaTaskResult):
```

Comment:
As we are using the json serializer of Kombu and not the pickle version I think this code can be removed as Kombu will not use the pickle module. In older version of Kombu the default was pickle but that has changed to json for security reasons.

https://docs.celeryproject.org/projects/kombu/en/stable/userguide/serialization.html

Reply:
The PSQ version still uses pickle though, and this method is independent of which task manager communication channel is being used.  Given that, I'll keep the pickle version here for now, and when we replace PSQ we can adjust as needed (hopefully removing the pickle check completely).